e2e/test/scenarios/sharing/sharing-reproductions.cy.spec.js

```diff
@@ -350,22 +351,21 @@ describe("issue 21559", { tags: "@external" }, () => {
 
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
     cy.findByText(q2Details.name).click();
-    cy.findByTestId("add-series-modal").button("Done").click();
+    cy.findByTestId("add-series-modal").within(() => {
+      // wait for elements to appear inside modal
+      chartPathWithFillColor("#A989C5").should("have.length", 1);
+      chartPathWithFillColor("#88BF4D").should("have.length", 1);
+
+      cy.button("Done").click();
+    });
 
     // Make sure visualization changed to bars
     chartPathWithFillColor("#A989C5").should("have.length", 1);
     chartPathWithFillColor("#88BF4D").should("have.length", 1);
 
     saveDashboard();
 
-    cy.icon("subscription").click();
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Email it").click();
-
-    cy.findByPlaceholderText("Enter user names or email addresses")
-      .click()
-      .type(`${admin.first_name} ${admin.last_name}{enter}`)
-      .blur(); // blur is needed to close the popover
+    openAndAddEmailsToSubscriptions([`${admin.first_name} ${admin.last_name}`]);
```

Comment:
sidebar can fail opening, `cy.wait(1000)` before it helped to fix it locally, but I don't like it